firebase-ai-friendly-meals/apple/FriendlyMeals/FriendlyMeals/Features/SuggestRecipe/SuggestRecipeViewModel.swift

```diff
@@ -0,0 +1,70 @@
+//
+// FriendlyMeals
+//
+// Created by Peter Friese on 02.07.25.
+// Copyright © 2025 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import FirebaseAI
+import SwiftUI
+
+@Observable
+class SuggestRecipeViewModel {
+  var isGenerating = false
+  var isPresentingRecipe = false
+
+  var ingredients = "Tomatoes, Basil, Mozarella"
```

Comment:
Note that if you fix this typo it needs to be fixed in the screenshot as well.

Reply:
Good shout, thanks!